authz/providers/azure/rbac/checkaccessreqhelper.go

```diff
@@ -27,12 +27,14 @@ import (
 )
 
 const (
-	AccessAllowedVerdict     = "Access allowed"
-	Allowed                  = "allowed"
-	AccessNotAllowedVerdict  = "User does not have access to the resource in Azure. Update role assignment to allow access."
-	namespaces               = "namespaces"
-	NotAllowedForNonAADUsers = "Access denied by Azure RBAC for non AAD users. Configure --azure.skip-authz-for-non-aad-users to enable access."
-	NoOpinionVerdict         = "Azure does not have opinion for this user."
+	AccessAllowedVerdict            = "Access allowed"
+	Allowed                         = "allowed"
+	AccessNotAllowedVerdict         = "User does not have access to the resource in Azure. Update role assignment to allow access."
+	namespaces                      = "namespaces"
+	NotAllowedForNonAADUsers        = "Access denied by Azure RBAC for non AAD users. Configure --azure.skip-authz-for-non-aad-users to enable access."
+	NoOpinionVerdict                = "Azure does not have opinion for this user."
+	SkipAuthzCheckNoOpinionVerdict  = "Azure does not have opinion for this user. If you are a non AAD user, --azure.skip-authz-for-non-aad-users must be set to true. If you are an AAD user, no opinion is usually seen for 'can-i' requests"
+	SkipAuthzCheckNotAllowedVerdict = "Access denied by Azure RBAC for non AAD users. Configure --azure.skip-authz-for-non-aad-users to enable access. If you are an AAD user, no opinion is usually seen for 'can-i' requests"
```

Comment:
This won't be no opinion. It would be denied. for AAD user, we should recommend to set Extra:oid parameter in kubeconfig.